### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mercadoa**
Updating the readme to point to the new Kadena Docs site and deprecate some of the information in the Wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,14 +31,9 @@ For additional information, press, and development inquires, please refer to the
...
-## Wiki
+## Docs
 
-The Chainweb wiki serves as a source of information that receives regular
-updates. You can find the chainweb Wiki
-[here](https://github.com/kadena-io/chainweb-node/wiki/), including a list of
-frequently asked questions regarding network information, how to explore blocks,
-diagnosing error messages, and more
-[here](https://github.com/kadena-io/chainweb-node/wiki/Chainweb-FAQ).
+The Kadena Docs site, which can be found [here](https://kadena-io.github.io/kadena-docs/) serves as a source of information about Kadena. You can find information about how to interact with the public chain, including how to get keys, view network activity, explore blocks, etc. here [here](https://kadena-io.github.io/kadena-docs/Public-Chain-Docs/).
```
Comment (fosskers) [inline README.md:36]:
```suggestion
The Kadena Docs site, which can be found [here](https://kadena-io.github.io/kadena-docs/) serves as a source of information about Kadena. You can find information about how to interact with the public chain, including how to get keys, view network activity, explore blocks, etc. [here](https://kadena-io.github.io/kadena-docs/Public-Chain-Docs/).
```

File: README.md [hunk 2]
```diff
@@ -196,6 +220,47 @@ Then, to run your node:
...
+You can get the cut height of any node by running the following:
+```
+$ https://<bootstrap-node-url>/chainweb/0.0/mainnet01/cut | python -m json.tool | jq '.height'
+```
+
+### Miscellaneous
+To find your public ip:
+```
+$ dig +short myip.opendns.com @resolver1.opendns.com
+$ dig TXT +short o-o.myaddr.l.google.com @ns1.google.com
```
Comment (larskuhtz) [inline README.md:261]: The following might be easier and doesn't introduce yet another tool:

```suggestion
$ curl 'https://api.ipify.org?format=text'
```

File: README.md [hunk 3]
```diff
@@ -172,7 +168,35 @@ To install a runnable binary to `~/.cabal/bin/`:
...
+- us-w2.chainweb.com
+- us-w3.chainweb.com
+- jp1.chainweb.com
+- jp2.chainweb.com
+- jp3.chainweb.com
+- fr1.chainweb.com
+- fr2.chainweb.com
+- fr3.chainweb.com
+
+## Configuring, running, and monitoring the health of a Chainweb Node
```
Comment (larskuhtz) [inline README.md:199]: Would it make sense to add a subsection about using the [docker image](https://hub.docker.com/repository/docker/kadena/chainweb-node) to run a node. It's not as flexible as running the binary directly. But it's an easy way to quickly get up and running (including initializing the database).

Comment (mercadoa) [reply README.md:199]: Its a good idea and something we should add, but we can do it a different PR

Comment (mightybyte) [reply README.md:199]: Let's do that in a follow-up PR.  I'd like to get this in so people know about the new kadena-docs site.

**Pull request merged**